source/Calamari.Shared/Integration/Scripting/WindowsPowerShell/Bootstrap.ps1

```diff
@@ -569,7 +569,12 @@ function Initialize-ProxySettings() {
 		if ($useDefaultProxy) {
 			# The system proxy should be provided through an environment variable, which has been used to initialize $proxyHost
 			if ($proxyUri -ne $null) {
-				$proxy = New-Object System.Net.WebProxy($proxyUri)
+				if($Env:OS -like "Windows*"){
```

Comment:
Would `if([System.Environment]::OSVersion.Platform -eq [System.PlatformID]::Win32NT){` be better?

Reply:
👍 yea good call, that's a much better check

Reply:
What about `if ($IsWindows)`? Only works on netcore, so it'd have to be 
```
if (-not (Test-Path variable:IsWindows) || $isWindows)
```